Week3/script.js

```diff
@@ -0,0 +1,273 @@
+/* *** String! *** */
+
+// 1.1
+
+let myString = "hello,this,is,a,difficult,to,read,sentence";
+
+console.log(myString);
+
+// 1.2
+
+console.log(myString.length);
+
+// 1.3 & 1.4
+
+console.log(myString.replace(/,/g, " "));
+
+/* ***Arrays!*** */
+
+let favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
+
+// 2.1
+
+favoriteAnimals.push("turtle");
+
+// 2.2
+
+console.log(favoriteAnimals);
+
+// 2.3
+
+favoriteAnimals.splice(1, 0, "meerkat");
+
+// 2.4
+
+console.log(
+  'favoriteAnimals will be: ["blowfish", "meerkat", "capricorn", "giraffe", "turtle"] '
+);
+
+// 2.5
+
+console.log(favoriteAnimals);
+
+// 2.6
+
+console.log("The array has a length of : 5");
+
+// 2.7
+
+favoriteAnimals.splice(3, 1);
+
+// 2.8
+
+console.log(favoriteAnimals);
+
+// 2.9
+
+console.log(favoriteAnimals.indexOf("meerkat"));
+
+// 2.10
+
+console.log("The item you are looking for is at index: 1");
+
+        /* *** More JavaScript *** */
+
+        // 1
+
+function threeArguments(a, b, c) {
+  var sum = a + b + c;
+
+  return sum;
+}
+
+        // 2
+
+function colorCar(car) {
+  return car;
+}
+
+let car = "a red car";
+
+console.log(colorCar(car));
+
+        // 3
+
+const person = {
+  name: "Mehmet Ekinci",
+  gender: "M",
+  adres() {
```

Comment:
I don't think this works as a property, did you try to run this part of the code?

Reply:
after your feedback I tried another thing. I hope it works.